mappings/net/minecraft/client/world/ClientWorld.mapping

```diff
@@ -115,6 +119,15 @@ CLASS net/minecraft/class_638 net/minecraft/client/world/ClientWorld
 	METHOD method_35753 putMapStates (Ljava/util/Map;)V
 		ARG 1 mapStates
 	METHOD method_35754 getMapStates ()Ljava/util/Map;
+	METHOD method_38534 runQueuedChunkUpdates ()V
+	METHOD method_38535 (Lnet/minecraft/class_2338;)I
+		ARG 1 pos
+	METHOD method_38536 enqueueChunkUpdate (Ljava/lang/Runnable;)V
```

Comment:
Hmm. The profiler section for method_38534 (runQueuedChunkUpdates) is called "chunk update queues", and there's "light update" below for calls to `LightingView#doLightUpdates`. I'm sure these two are different but let me see.

(intermediary bc i forgot to compile named map and opening engima requires gigabytes of ram)

```java
/* class_761 method_22710 */
    lv.method_15405("chunk_update_queue");
    this.field_4085.method_38534();
    lv.method_15405("light_updates");
    this.field_4085.method_2935().method_12130().method_15516(...);
```

Reply:
Given this is an update queue to be run on the world renderer, it can be any type of chunk update (such as used by mods); it is not required to be a light update.

Reply:
Yep, keeping that.